consensus/istanbul/backend/backend.go

```diff
@@ -196,7 +196,7 @@ func (sb *Backend) IsProxy() bool {
 }
 
 func (sb *Backend) IsProxiedValidator() bool {
-	return sb.proxyNode != nil
+	return sb.proxyNode != nil && sb.proxyNode.peer != nil
```

Comment:
Are there potential ramifications of having a proxied validator not connected to their proxy act as if they're not proxied? cc @kevjue